client/src/components/pages/ChannelCreate.tsx

```diff
@@ -351,11 +364,12 @@ const ChannelCreate: FC = () => {
         </TouchableOpacity>
       ),
     });
-  }, [commitChannel, navigation, selectedUsers]);
+  }, [commitChannel, navigation, selectedUsers, theme]);
 
   const searchArgs: ChannelCreateFriendsPaginationQueryVariables = {
     first: ITEM_CNT,
     searchText: debouncedText,
+    includeMe: true,
```

Comment:
Do we need to search for self when creating a channel?

Reply:
It's a matter of choice, I made it possible that a user can also create  the self channel in the `ChanneCreate` by selecting the user's own profile but if it is regarded as unnecessary, this variable should be deleted. 
how do you think about this? @hyochan

Reply:
I think this design is similar to `KakaoTalk` and IMO, it looks great